config/ckan_api.php

```diff
@@ -0,0 +1,2 @@
+CKAN_API_URL=""
+CKAN_API_KEY=""
```

Comment:
this needs to come from the database not from the .ENV since it's set up by each user

Reply:
That is going to be more complex then because I don't think this client supports that but I'll look.

Reply:
cool. For context: this key is the one the user provides and you save in the user settings hdx.api_key config value